contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -1020,12 +1020,13 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
     }
     delete s_billingTokens;
 
+    PayoutMode mode = s_payoutMode;
     for (uint256 i = 0; i < billingTokens.length; i++) {
       IERC20 token = billingTokens[i];
       BillingConfig memory config = billingConfigs[i];
 
       // if LINK is a billing option, payout mode must be ON_CHAIN
-      if (address(token) == address(i_link) && s_payoutMode == PayoutMode.OFF_CHAIN) {
+      if (address(token) == address(i_link) && mode == PayoutMode.OFF_CHAIN) {
```

Comment:
I think this change probably _increases_ gas, fwiw, since we read the thing into memory now